### ISSUE TITLE
Uncaught ReferenceError: cocoSsd is not defined

### ISSUE THREAD

**Issue by paapu88** [tensorflow/tfjs#902]
#### TensorFlow.js version
https://cdn.jsdelivr.net/npm/@tensorflow/tfjs
#### Browser version
Chrome70
#### Describe the problem or feature request
When running the script 
via Script Tag  ( https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd#via-script-tag )
on my browser
I get error 
Uncaught ReferenceError: cocoSsd is not defined

Terveisin, Markus

**Comment by dsmilkov** [comment]
@pyu10055 can you release a new version of the coco-ssd model? I think we didn't release after you fixed the rollup config.

**Comment by pyu10055** [comment]
@paapu88 for now, you can use following scripts before we push the new release. thanks.

```
<!-- Load TensorFlow.js. This is required to use coco-ssd model. -->
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@0.12.7"> </script>
<!-- Load the coco-ssd model. -->
<script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/coco-ssd"> </script>

<!-- Replace this with your image. Make sure CORS settings allow reading the image! -->
<img id="img" src="cat.jpg"/>

<!-- Place your code in the script tag below. You can also use an external .js file -->
<script>
  // Notice there is no 'import' statement. 'cocoSsd' and 'tf' is
  // available on the index-page because of the script tag above.

  const img = document.getElementById('img');

  // Load the model.
  window['coco-ssd'].load().then(model => {
    // detect objects in the image.
    model.detect(img).then(predictions => {
      console.log('Predictions: ', predictions);
    });
  });
</script>
```

**Comment by pyu10055** [comment]
Just released a new version for coco-ssd, v0.1.1 to fix this issue.

**Comment by paapu88** [comment]
Thank's,
I get the cat prediction now with the script below
BUT: I need to run chromium with 
`
 /c/Program\ Files\ \(x86\)/Google/Chrome/Application/chrome.exe --disable-web-security --user-data-dir
`
in my windows10-git git shell (from https://gitforwindows.org/ ).
Is there no easier way to do this? If not you should DOCUMENT this.
Terveisin, Markus 


 (from [github](https://github.com/tensorflow/tfjs-models/tree/master/coco-ssd#via-script-tag))

```
<!-- Load TensorFlow.js. This is required to use coco-ssd model. -->
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"> </script>
<!-- Load the coco-ssd model. -->
<script src="https://cdn.jsdelivr.net/npm/@tensorflow-models/coco-ssd"> </script>

<!-- Replace this with your image. Make sure CORS settings allow reading the image! -->
<img id="img" src="cat.jpg"/>

<!-- Place your code in the script tag below. You can also use an external .js file -->
<script>
  // Notice there is no 'import' statement. 'cocoSsd' and 'tf' is
  // available on the index-page because of the script tag above.

  const img = document.getElementById('img');

  // Load the model.
  cocoSsd.load().then(model => {
    // detect objects in the image.
    model.detect(img).then(predictions => {
      console.log('Predictions: ', predictions);
    });
  });
</script>
```
